.readthedocs.yaml

```diff
@@ -19,8 +19,7 @@ formats:
 
 # Optionally set the version of Python and requirements required to build your docs
 python:
-  version: 3.7
+  version: 3.8
```

Comment:
I don't think this was necessary but it seemed like a good idea

Reply:
I think so too!